src/ert/scheduler/lsf_driver.py

```diff
@@ -266,6 +266,7 @@ async def submit(
             retry_interval=self._sleep_time_between_cmd_retries,
         )
         if not process_success:
+            self._job_error_message[iens] = process_message
             raise RuntimeError(process_message)
```

Comment:
I thought this would be logged regardless. We run this coroutine in an event loop from async_utils.py, so it will log any raised errors once it is done.
But maybe it's good to log it once more just to be sure.

Reply:
For some reason the `RuntimeError` can't be found in logs.

Reply:
Not under jobqueue logs either?